File: BlaBlaCar.DAL/Entities/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BlaBlaCar.DAL.Entities
{
    public class Car: BaseEntity
    {
        public string ModelName { get; set; }
        public string RegistNum { get; set; }
        public CarType CarType { get; set; }
        public Guid UserId { get; set; }
        public Status CarStatus { get; set; }
        public ICollection<CarDocuments> CarDocuments { get; set; }
        public ApplicationUser User { get; set; }
        public ICollection<Seat> Seats { get; set; }
        public ICollection<Trip> Trips { get; set; }
    }
}

File: web-client/src/app/interfaces/car.ts
import { CarType } from "../enums/car-type";
import { CarStatus } from "./car-status";
import { SeatModel } from "./seat";

export interface CarModel {
    id: number;
    modelName: string;
    registNum:string;
    carType: CarType;
    seats: SeatModel[];
    techPassport: string;
    carStatus:CarStatus;
}
